src/it/parallel-modules/.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-T10
```

Comment:
wow, I already forgot this issue...

I think it was and I manage to reproduce the problem locally once I included the tests